src/components/map/map.jsx

```diff
@@ -9,25 +8,38 @@ class Map extends PureComponent {
   }
 
   componentDidMount() {
-    const {offers, leaflet} = this.props;
+    const {activeOffer, offers, leaflet} = this.props;
     const city = [52.38333, 4.9];
     const icon = leaflet.icon({
       iconUrl: `img/pin.svg`,
-      iconSize: [30, 40]
+      iconSize: [30, 40],
+    });
+    const activeIcon = leaflet.icon({
+      iconUrl: `img/pin-active.svg`,
+      iconSize: [30, 40],
     });
     const zoom = 12;
     const map = leaflet.map(`map`, {
       center: city,
       zoom,
       zoomControl: false,
-      marker: true
+      marker: true,
+      maxZoom: zoom,
+      minZoom: zoom,
     });
     map.setView(city, zoom);
     leaflet
             .tileLayer(`https://{s}.basemaps.cartocdn.com/rastertiles/voyager/{z}/{x}/{y}{r}.png`, {
               attribution: `&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors &copy; <a href="https://carto.com/attributions">CARTO</a>`
             })
             .addTo(map);
+
+    if (activeOffer) {
+      leaflet
+            .marker(activeOffer.coordinates, {activeIcon})
```

Comment:
```suggestion
            .marker(activeOffer.coordinates, {icon: activeIcon})
```

это решит проблему с активным маркером. Думаю, не надо объяснять почему :)